.travis.yml

```diff
@@ -27,6 +27,7 @@ matrix:
 
     - { python: "3.8", env: TOXENV=py38-django22-postgres }
     - { python: "3.8", env: TOXENV=py38-django30-postgres }
+    - { python: "3.8", env: TOXENV=py38-django31-postgres }
```

Comment:
maybe worth adding  `{ python: "3.8", env: TOXENV=py38-django31-mysql_native_json }` as well?

Reply:
Oh yeah.

Reply:
Looks like 3.1 supports it on sqlite as well so I added a separate matrix for it.